internal/metadata/clusters/azure_ks.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: move2kube.io/v1alpha1
+kind: ClusterMetadata
+metadata:
+  name: AZURE-KS
```

Comment:
@akhil-ghatiki Thanks. This is a great fix. Can we rename it as Azure-AKS, since Azure is not an acronym (hence non caps) and AKS will help users relate to it.

Reply:
Changed the name and pushed it.